app/src/test/java/com/simplemobiletools/calculator/MainActivityTest.kt

```diff
@@ -52,7 +52,7 @@ class MainActivityTest {
         var result  = activity.getResult()
 
         assertEquals("1", result)
-        checkFormula("5-4")
```

Comment:
Pretty crazy that they both look the same in a monospace font. It's better now as it displays the proper minus sign on the calculator

Reply:
I agree @jusleg thats quite the point to make!